management-cluster-access/README.md

```diff
@@ -0,0 +1,119 @@
+# Ensure no single point of failure in management cluster access
+
+## Access to MCs for giant swarm staff in vintage vs the CAPI product
+
+Our default access method to management clusters is now single-sign-on.
+For Giant Swarm staff this is done via `dex` using github as the _only_ identity provider. 
+
+Our vintage product also offers the possibility to create a client certificate to access the management cluster.
+This means that in the event of github or `dex` being unavailable, we do not get locked out of the management cluster.
+
+For pure CAPI installations on the other hand, the only fallback authentication method is a single static kubeconfig that is stored in lastpass for emergency access.
+This poses a real risk to get locked out of the management cluster in case of problems with OIDC.
```

Comment:
Since there is a fallback kubeconfig in the password manager, this is not really true, is it? However, access is somewhat inconvenient.